tutor/plugins/openedx.py

```diff
@@ -142,3 +142,63 @@ def is_directory_mounted(image_name: str, dirname: str) -> bool:
 hooks.Filters.ENV_TEMPLATE_VARIABLES.add_item(
     ("iter_mounted_directories", iter_mounted_directories)
 )
+
+
+@hooks.Filters.LMS_WORKER_COMMAND.add(priority=hooks.priorities.HIGH)
+def _add_default_lms_worker_parameters(
+    command: list[str]
+) -> list[str]:
+    command = [
+        "celery",
+        "worker",
+        "--app=lms.celery",
+        "--loglevel=info",
+        "--hostname=edx.lms.core.default.%%h",
```

Comment:
Actually, this should be "%h". The extra "%" is a [very old artefact](https://github.com/overhangio/tutor/commit/e357e763d703a4c8ca0b3f7a79a7be675b296297) from the time I copied the command string from ansible templates. As a result, celery workers have an incorrect name, such as "celery@edx.lms.core.default.%b92a7a891007". Notice the extra "%".

This is the right time to fix this 7-year-old bug :)